src/clients/HubPoolClient.ts

```diff
@@ -500,6 +500,10 @@ export class HubPoolClient extends BaseAbstractClient {
       }
     ) as ExecutedRootBundle;
 
+    return this.getRunningBalanceForToken(l1Token, executedRootBundle);
+  }
+
+  getRunningBalanceForToken(l1Token: string, executedRootBundle: ExecutedRootBundle): TokenRunningBalance {
```

Comment:
Nit: should we make this public

```suggestion
  public getRunningBalanceForToken(l1Token: string, executedRootBundle: ExecutedRootBundle): TokenRunningBalance {
```